spec/serializers/firm_serializer_spec.rb

```diff
@@ -40,6 +40,14 @@
       expect(subject[:investment_sizes]).to eql(firm.investment_size_ids)
     end
 
+    it 'exposes `advises_on_investments`' do
+      expect(subject[:advises_on_investments]).to be
+    end
+
+    it 'exposes `investment_transfers`' do
+      expect(subject[:investment_transfers]).to be
+    end
+
```

Comment:
Is it already covered both cases? I tried to say e.g: if pension_transfer_percent is zero the API will returns false for that field. Or if investment_size_ids is empty will returns false. That both scenarios are covered?